src/components/header/navbar.tsx

```diff
@@ -0,0 +1,44 @@
+import { NavLink } from "react-router";
+import { ABOUT_PAGE, HOME_PAGE, PRODUCT_PAGE } from "@/routing/Links";
+import * as styles from "./navbar.m.scss";
+
+export default function Navbar() {
+  return (
+    <header className={styles.navBar}>
+      <nav>
+        <NavLink
+          to={HOME_PAGE}
+          className={styles.navButton}
+          end
+          style={({ isActive }) => ({
```

Comment:
Please move the active styling from the inline style to a CSS class. You can use the `clsx` package https://www.npmjs.com/package/clsx to combine conditional classes.